src/api/v3/single_page.py

```diff
@@ -0,0 +1,41 @@
+"""
+View to return a JSON representation of a single page. The page can
+be selected via the id or the permalink.
+"""
+from django.http import JsonResponse, HttpResponse
+from django.shortcuts import get_object_or_404
+
+from cms.models import Region
```

Comment:
```suggestion
from cms.models import Region, Page
```
This is needed if you apply my suggestion below.